src/test/resources/Features/OCD - 2262-RemoveDeprecatedApiPostCallsOfPutOrDelete.feature

```diff
@@ -0,0 +1,50 @@
+Feature: OCD-2262 - Remove API POST calls that were deprecated in favor of PUT/DELETE 
+    This test checks whether deprecated calls are removed from CHPL API webpage
+
+@Regression
+Scenario: CHPL API page loads and shows correct title 
+    Given I am on CHPL API page 
+    Then CHPL API page title should be "CHPL API" 
+
+Scenario Outline: 
```

Comment:
When is this outline run? It's not tagged with anything, and looks very similar to the Acceptance scenario below